deploy/default-config.yaml

```diff
@@ -0,0 +1,10 @@
+kind: DevWorkspaceOperatorConfig
+apiVersion: controller.devfile.io/v1alpha1
+metadata:
+  name: devworkspace-operator-config
```

Comment:
```suggestion
  name: devworkspace-operator-config
  namespace: ${NAMESPACE}
```

Reply:
Also, I got my routing suffix wrongly assigned
![Screenshot_20210920_144601](https://user-images.githubusercontent.com/5887312/133996978-30039921-5ea2-4621-9883-ffd02a827810.png)

Reply:
Ahh yes, previously, on OpenShift we would always overwrite `clusterHostSuffix` from a route, even if the configmap contained a different value. With the current changes, we only create the test route when `clusterHostSuffix` is unset (which doesn't happen with the makefile, since it creates a config).

I'll think about how best to handle this -- should we just assume OpenShift users want to use the default fill for `route.spec.host`?

Reply:
I've updated the Makefile to not set a default `ROUTING_SUFFIX` anymore. On OpenShift, this should result in the config getting an empty value, which would be auto-filled by the controller. On Kubernetes, we auto-detect it in the makefile for minikube and print a warning if it's empty.